### PR TITLE
[FIX] Display total validations on home page

### PR DESCRIPTION
**Pull request by stevenzeiler**

### CODE REVIEW

**Comment by stevenzeiler** [comment]
:+1:

File: scripts/services/Validators.js [hunk 1]
```diff
@@ -11,38 +25,64 @@ angular.module('validatorsApp').factory('Validators',['$http', function($http) {
...
-      })  
+    if (combined) {
+      return Promise.resolve(combined)
     } else {
-      return fetch()
+      return fetchValidations().then(function(validations) {
+
+        return fetchValidators().then(function(validators) {
+
+          combined = sortByValidations(reduceValidators(validators, validations))
```
Comment (wilsonianb) [inline scripts/services/Validators.js:42]: Should we go ahead and switch to sorting by correlation coefficient?

Comment (wilsonianb) [inline scripts/services/Validators.js:42]: Oh, we're not using the /reports endpoint yet

Comment (stevenzeiler) [inline scripts/services/Validators.js:42]: Let's sort by correlation coefficient once it is available via the API.

Comment (wilsonianb) [inline scripts/services/Validators.js:42]: https://github.com/ripple/validator-registry-api/pull/51

**Comment by wilsonianb** [comment]
Domain is not being displayed on validator historical performance page

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**